### PR TITLE
[IMP] sale: add product in sale report tree

### PR DESCRIPTION
**Pull request by niyasraphy**
before this commit, in the sale.report tree
view, the product field was not added.

* sales -> reporting -> switch to pivot
* click on any row to open the tree

after this commit, the product field will
be added in the sale.report tree view.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by niyasraphy** [comment]
hi @Feyensv ,
if you have time, can you review this pr.
thanks

File: addons/sale/report/sale_report_views.xml [hunk 1]
```diff
@@ -33,6 +33,7 @@
                 <field name="order_reference" optional="show"/>
                 <field name="partner_id" optional="hide"/>
                 <field name="user_id" optional="show" widget="many2one_avatar_user"/>
+                <field name="product_id" string="Product" optional="show"/>
```
Comment (dalaOdoo) [inline addons/sale/report/sale_report_views.xml:36]:
```suggestion
                <field name="partner_id" optional="hide"/>
                <field name="product_id" string="Product" optional="show"/>
                <field name="user_id" optional="show" widget="many2one_avatar_user"/>
```

**Review by dalaOdoo — CHANGES_REQUESTED** [review]
I think we should not separate sales person and sales team so i would suggest to put the product before.

**Review by Feyensv — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**